jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -53,6 +59,10 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 Tracer.startSpan(operation, spanId); // caller's span is this span's parent.
             }
         }
+
+        // Give SLF4J appenders access to the trace id
+        // TODO(rfink) We should use putCloseable; when and how can we remove it though? There is no filter chain.
+        MDC.put(MDC_KEY, Tracer.getTraceId());
```

Comment:
(this allows picking things up in the Access logger, assuming it's not asynchronous or logback fixes the bug I submitted a patch for)